<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addEntryWindow</class>
 <widget class="QWidget" name="addEntryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1880</width>
    <height>1020</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 26, 26);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>700</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <property name="rightMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1220</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
         <property name="leftMargin">
          <number>19</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLineEdit" name="monthEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>480</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>18</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 7px;
color: rgb(255, 255, 255);
font: 700 18pt &quot;Segoe UI&quot;;
qproperty-alignment: AlignCenter;</string>
           </property>
           <property name="placeholderText">
            <string>Monat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="yearEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(74, 74, 74);
border-radius: 7px;
qproperty-alignment: AlignCenter;
color: rgb(255, 255, 255);
font: 700 18pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="placeholderText">
            <string>Jahr</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollBar:vertical {
    border: none;
    background: none;
    width: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::add-page:vertical {
	background: rgb(171, 255, 185);
}

QScrollBar::handle:vertical {
	background: rgb(53, 53, 53);
    min-height: 20px;
}

QScrollBar::add-line:vertical {
    background: none;
}

QScrollBar::sub-line:vertical {
    background: none;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1583</width>
           <height>896</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="cellLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QPushButton" name="homeButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: rgb(43, 43, 43);
border-radius: 25px;
}

QPushButton:hover {
	background-color: rgb(38, 38, 38);
}

QPushButton:clicked {
	background-color: rgb(34, 34, 34);
}

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../assets/recource.qrc">
          <normaloff>:/assets/home.png</normaloff>:/assets/home.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>380</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="confimAllButton">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
font: 700 16pt &quot;Segoe UI&quot;;
	background-color: rgb(57, 57, 57);
color: rgb(255, 255, 255);
border-radius: 10px;

}

QPushButton:hover{
	
	background-color: rgb(152, 235, 143);
}
QPushButton:pressed{
	
	background-color: rgb(120, 229, 128);
}</string>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
        <property name="icon">
         <iconset resource="../../../assets/recource.qrc">
          <normaloff>:/assets/confirmCheck.png</normaloff>:/assets/confirmCheck.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../assets/recource.qrc"/>
 </resources>
 <connections/>
</ui>
